monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/PluginTable.tsx

```diff
@@ -134,7 +161,7 @@ const PluginTable = (props) => {
         ? loadingMessage
         : <XDataGrid columns={columns}
                      rows={[...rows]}
-                     rowHeight={'25px'}
+                     getRowHeight={() => 'auto'}
```

Comment:
There's no need to pass this prop to get an auto-height as the XDataGrid handles it